src/3a.js

```diff
@@ -0,0 +1,86 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const myData = require("./data");
+const sha256 = require('js-sha256');
+const app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
```

Comment:
Use newlines to group different import types
```suggestion
const express = require("express");
const bodyParser = require("body-parser");
const sha256 = require('js-sha256');

const myData = require("./data");

const app = express();

app.use(bodyParser.json());
app.use(bodyParser.urlencoded({extended: false}));
```